zaza/openstack/charm_tests/glance_simplestreams_sync/tests.py

```diff
@@ -125,3 +125,21 @@ def _check_local_product_streams(expected_images):
         _check_local_product_streams(expected_images)
 
         logging.debug("Local product stream successful")
+
+
+class GlanceSimpleStreamsSyncWithPropertiesTest(GlanceSimpleStreamsSyncTest):
+
+    @classmethod
+    def setUpClass(cls):
+        """Run class setup for running glance simple streams sync tests."""
+        super(GlanceSimpleStreamsSyncWithPropertiesTest, cls).setUpClass()
+        zaza_model.set_application_config('glance-simplestreams-sync', {'set_latest_property': 'true'})
+
+    # TODO(guimalufb) test if the latest property gets removed from old images
+    def test_200_check_image_latest_property(self):
+        """Verify that a property inclusion has been done"""
+        logging.debug("Checking image latest property")
+
+        _filter_properties = {'filters': {'latest': 'true'}}
```

Comment:
Not sure why you need an underscore variable here; it's already 'private' to the function?